src/adapter/src/coord.rs

```diff
@@ -754,7 +760,7 @@ impl<S: Append + 'static> Coordinator<S> {
                 }
                 // `tick()` on `Interval` is cancel-safe:
                 // https://docs.rs/tokio/1.19.2/tokio/time/struct.Interval.html#cancel-safety
-                _ = advance_timelines_interval.tick() => Message::AdvanceTimelines,
+                _ = advance_timelines_interval.tick() => Message::GroupCommitInitiate,
```

Comment:
If we wanted to keep things moving, we could tick this each time the group commit completes.

Reply:
I wouldn't be against that. My one fear is that we'd end up clogging the Coordinator up somehow, but since append is now done asynchronously it should be fine?